app/components/events/view/export/download-zip.js

```diff
@@ -3,4 +3,9 @@ import Ember from 'ember';
 const { Component } = Ember;
 
 export default Component.extend({
+  actions: {
+    startGeneration() {
```

Comment:
@niranjan94 Do you want me to move all the logic from my controller to the component itself?